libs/ui-lib/lib/ocm/components/clusterWizard/OperatorsStep.tsx

```diff
@@ -21,10 +21,10 @@ export const OperatorsStep = (props: ClusterOperatorProps) => {
         <ClusterWizardStepHeader>Operators</ClusterWizardStepHeader>
       </StackItem>
       <StackItem>
-        <CnvCheckbox {...props} />
+        <CnvCheckbox {...props} showMessageForLvms={isVersionEqualsOrMajorThan4_15 || isSNO} />
       </StackItem>
       <StackItem>
-        <MceCheckbox />
+        <MceCheckbox showMessageForLvms={isVersionEqualsOrMajorThan4_15 || isSNO} isSNO={isSNO} />
```

Comment:
I'd only pass a boolean `isVersionEqualsOrMajorThan4_15` prop to these components and leave the logic for what message to show to those components. Doing checks of `isSNO` here and then again inside of those components is either redundant, very confusing or both.